build.sbt

```diff
@@ -30,9 +30,11 @@ scalacOptions <++= scalaVersion map { sv =>
     "-feature",
     "-language:postfixOps",
     "-language:implicitConversions",
-    "-language:higherKinds"
+    "-language:higherKinds",
+    "-target:jvm-1.8"
```

Comment:
Let's leave changes to scala 2.10 to the other 1.0.x  branch (and we don't have to update that one). Basically this line is unnecessary.